app/electron/main.js

```diff
@@ -133,8 +140,34 @@ function createWindow () {
   mainWindow.on('closed', () => {
     mainWindow = null;
   });
+  mainWindow.once('ready-to-show', () => {
+    autoUpdater.checkForUpdatesAndNotify();
+  })
+}
+
+autoUpdater.autoDownload = true
+
+function sendStatusToWindow(text) {
+  log.info(text);
+  mainWindow.webContents.send('message', text);
 }
 
+autoUpdater.on('update-not-available', (info) => {
+  sendStatusToWindow('Update not available.');
```

Comment:
You're using a period here, but not in the other messages. Please remove it from here.